boards/arm/legend25/doc/index.rst

```diff
@@ -0,0 +1,130 @@
+| UART      | on-chip    | serial                              |
++-----------+------------+-------------------------------------+
+| SPI flash | on-chip    | spi_nor                             |
++-----------+------------+-------------------------------------+
+| LED strip | on-chip    | ws2812                              |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | usb                                 |
++-----------+------------+-------------------------------------+
+
+
+Connections and IOs
+===================
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PB6     | I2C1            | I2C1 SCL (HDD only)        |
++---------+-----------------+----------------------------+
+| PB7     | I2C1            | I2C1 SDA (HDD only)        |
++---------+-----------------+----------------------------+
+| PA10    | UART            | USART0 RX                  |
++---------+-----------------+----------------------------+
+| PA9     | UART            | USART0 TX                  |
++---------+-----------------+----------------------------+
+| PB0     | PWM             | Activity LED (HDD only)    |
++---------+-----------------+----------------------------+
+| PB12    | SPI2            | SPI2 Enable                |
++---------+-----------------+----------------------------+
+| PB13    | SPI2            | SPI2 Clock                 |
++---------+-----------------+----------------------------+
+| PB14    | SPI2            | SPI2 MISO                  |
++---------+-----------------+----------------------------+
+| PB15    | SPI2            | SPI2 MOSI                  |
++---------+-----------------+----------------------------+
+| PA7     | LED strip       | SPI1 MOSI                  |
++---------+-----------------+----------------------------+
+| PA12    | USB             | USB DM (SSD only)          |
++---------+-----------------+----------------------------+
+| PA13    | USB             | USB DP (SSD only)          |
++---------+-----------------+----------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+The STM32F070cb MCU can be flashed by connecting an external debug probe to the
+SWD port (on-board 4-pin header). In the default OpenOCD configuration, the
+ST Link interface is selected. You may need to replace it with the interface of
+your debug probe.
+
+Once the debug probe is connected to both the Legend board and your host
+computer, then you can simply run the ``west flash`` command to write a firmware
+image into flash.
+
+Debugging
+=========
+
+Please refer to the `Flashing`_ section and run the ``west debug`` command
+instead of ``west flash``.
+
+References
+**********
+
+.. target-notes::
+
+.. _STM32F070 reference manual:
+   http://www.st.com/resource/en/reference_manual/dm00031936.pdf
```

Comment:
I didn't build the docs for this PR, but is this rendering the way you want it to?